automation/emit_coverage_info.sh

```diff
@@ -27,20 +28,16 @@ for manifest in $(cargo metadata --no-deps --format-version 1 | jq -r '.packages
     package=$(dirname "$manifest")
     package=$(basename "$package")
     echo "## no-default-features test for package $package (manifest @ $manifest)"
-    cargo +nightly test --manifest-path "$manifest" --no-default-features
+    cargo +nightly test --manifest-path "$manifest" --all-targets --no-default-features --no-fail-fast
 done
 
-for bench_pkg in $(cargo metadata --format-version 1 --no-deps | jq -r '.packages[]  | select(.targets[] | select(.kind[] | contains("bench"))) | .name' | uniq); do
-    cargo +nightly test -p "$bench_pkg" --benches
-done
+# for bench_pkg in $(cargo metadata --format-version 1 --no-deps | jq -r '.packages[]  | select(.targets[] | select(.kind[] | contains("bench"))) | .name' | uniq); do
```

Comment:
I think --all-targets covers this but we'll see.

Note that --all-targets does not run doctests :/ https://github.com/rust-lang/cargo/issues/6669